datadog/resource_datadog_logs_metric.go

```diff
@@ -0,0 +1,318 @@
+package datadog
+
+import (
+	"fmt"
+
+	datadogV2 "github.com/DataDog/datadog-api-client-go/api/v2/datadog"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceDatadogLogsMetric() *schema.Resource {
+	return &schema.Resource{
+		Description: "Resource for interacting with the logs_metric API",
+		Create:      resourceDatadogLogsMetricCreate,
+		Read:        resourceDatadogLogsMetricRead,
+		Update:      resourceDatadogLogsMetricUpdate,
+		Delete:      resourceDatadogLogsMetricDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceDatadogLogsMetricImport,
+		},
+		Schema: map[string]*schema.Schema{
+
+			"compute": {
+				Type:        schema.TypeMap,
```

Comment:
TypeMap can't have a schema as its `Elem` - https://www.terraform.io/docs/extend/schemas/schema-types.html#typemap
For these cases its recommended to do a TypeList of MaxItems: 1